cmd/ir2proxy/main.go

```diff
@@ -70,12 +71,21 @@ func run() int {
 		// so filter it out.
 		// See https://github.com/projectcontour/ir2proxy/issues/8 for more explanation here.
 		outputYAML = bytes.ReplaceAll(outputYAML, []byte("  creationTimestamp: null\n"), []byte(""))
-		fmt.Printf("---\n%s", outputYAML)
+		outputWarnings := commentedWarnings(warnings)
+		fmt.Printf("---\n%s%s", outputWarnings, outputYAML)
 	}
 
 	return 0
 }
 
+func commentedWarnings(warnings []string) string {
+	for index, warning := range warnings {
+		warnings[index] = "# " + strings.ReplaceAll(warning, ". ", ".\n# ")
```

Comment:
This seems a bit fragile. Maybe we want to consider a Warning object that has multiple lines. Then we could join it for the single-line output and prefix it for the YAML.

Reply:
That's a good idea. I'll merge this one, then log an issue.